python/lsst/rapid/analysis/utils.py

```diff
@@ -217,3 +221,47 @@ def checkRubinTvExternalPackages(exitIfNotFound=True, logger=None):
 
     if exitIfNotFound and (not hasGoogleStorage or not hasEfdClient):
         exit()
+
+
+def getExpPositionOffset(exp1, exp2):
+    # need the exps if we want altAz because we need the observation times
+    wcs1 = exp1.getWcs()
+    wcs2 = exp2.getWcs()
+    p1 = wcs1.getSkyOrigin()
```

Comment:
The doc for `getSkyOrigin` says: 
```
Returns CRVAL. This need not be the centre of the image. 
```
I think you'll want to define more precisely the position you're comparing.